README.md

```diff
@@ -112,3 +163,33 @@ export INMANTA_MODULE_REPO=https://USER:LICENSE_TOKEN@modules.inmanta.com/git/in
     pytest tests
 ```
 
+### Deploy a local orchestrator
+
+It is possible to deploy an orchestrator locally and run the tests against it.  The orchestrator will be deployed as a container, using docker.  Here are the prerequisites in order to make it work:
```

Comment:
Could you add the exact/minimal command to use this to improve copy-pastability.

Reply:
I added more commands, is it better?